src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyStore.tsx

```diff
@@ -132,55 +132,43 @@ const KeyStoreFlow = () => {
     }
   };
 
+  const sendTagManagerEvent = (category: string, action: string, label: string) => {
+    GoogleTagManager.getInstance().sendEvent({ category, action, label });
+  };
+
   const RemoveButton = () => <Grid ref={removeButtons} onClick={removeFile} className={classes.Remove}>Remove</Grid>;
 
   const submitHandler = async () => {
     applicationStore.setIsLoading(true);
     setTimeout(async () => {
       try {
-      await validatorStore.extractKeyStoreData(keyStorePassword);
-      isDeposited;
-      const deposited = true; // await isDeposited();
-      applicationStore.setIsLoading(false);
-      validatorStore.registrationMode = 1;
-      slashingWarningNavigate[`${processStore.secondRegistration}`]();
+        await validatorStore.extractKeyStoreData(keyStorePassword);
+        // TODO fix this dummy value.
+        const deposited = true; // await isDeposited();
```

Comment:
Create task for yourself with reference to this line

Reply:
@alexstolr

Reply:
https://blox-ensemble.monday.com/boards/4755515647/pulses/5751878443